.eslintrc.js

```diff
@@ -21,33 +23,52 @@ module.exports = {
     '**/ios/main.jsbundle',
     '**/stylelint.config.js',
     '**/dist/',
-    '**/lib-dist/'
+    '**/lib-dist/',
+    '.eslintrc.js'
   ],
   rules: {
-    'arrow-body-style': 'error',
-    'arrow-parens': ['error', 'as-needed'],
+    // Overly strict rules (for now)
+    'class-methods-use-this': 'off',
+    'default-case': 'off',
```

Comment:
Maybe this is also really helpful, as having no default can lead to unexpected results, when not thinking about the default case. Having this rule enforces to think about it.

Reply:
We already have `'@typescript-eslint/no-use-before-define': 'error'` which does the same with typescript support. This rule also throws the error all the time:
> 'React' was used before it was defined  no-use-before-define

Reply:
Will add https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/switch-exhaustiveness-check.md which only enforces necessary switch statements. I don't think it makes sense being forced to add unnecessary defaults that are not possible to be executed.